src/app/common/utils.ts

```diff
@@ -292,3 +315,46 @@ export function removeTrailingNullCharacters(s: string) {
 export function removeMinusSign(value: string) {
   return value.replace('-', '');
 }
+
+export function filterTokens<T>({
+  tokens,
+  filter = 'all',
+  getTokenIdentifier,
+}: {
+  tokens: T[];
+  filter: AssetFilter | Sip10CryptoAssetFilter;
+  getTokenIdentifier: (token: T) => string;
+}): T[] {
+  const accountIndex = useCurrentAccountIndex();
```

Comment:
This won't work as you are using React Hooks inside of a standard JS function. 

If you want to use hooks then `filterTokens` also needs to be a hook

Do you have`eslint` setup? To help you during development you can run `pnpm lint:eslint` to check for errors. 

![Screenshot 2024-07-02 at 12 49 14](https://github.com/leather-io/extension/assets/2938440/ed32a26f-54c8-4f5e-8592-52f90eee55a7)

Reply:
thanks was using had a really outdated eslint without realizing

Reply:
No problem, thanks for your contributions! 👏